### ISSUE TITLE
(win) After uninstall I still get: "The Webcamoid virtual camera service service terminated unexpectedly..."

### ISSUE THREAD

**Issue by samozrejme** [webcamoid/webcamoid#428]
After I've uninstall Webcamoid I still get Errors in the Even Viewer on Windows.

Event Viewer - Windows Logs - System:
_The Webcamoid virtual camera service service terminated unexpectedly.  It has done this 13 time(s).  The following corrective action will be taken in 5000 milliseconds: Restart the service._

This error appears exactly every 30 minutes. The number in "It has done this _n_ time(s)." changes a bit. My system freezes for a second during this event.

I have also tried reinstalling the current version of Webcamoid and unistalling it with the uninstall.exe, but the error is still happening.

Please help me remove all of the Webcamoid files of my system.
And maybe please make sure the uninstall tool does its job as it is probably supposed to.
Thx

**Comment by hipersayanX** [comment]
>Please help me remove all of the Webcamoid files of my system.

Just click **Remove All** to remove the virtual camera, that should fix the problem.

![Screenshot_20210410_235057](https://user-images.githubusercontent.com/1249885/114290711-e7fecd80-9a57-11eb-8f10-c1a91e3b74e9.png)

>And maybe please make sure the uninstall tool does its job as it is probably supposed to.

This bug will be fixed in Webcamoid 9.

**Comment by samozrejme** [comment]
> > Please help me remove all of the Webcamoid files of my system.
> 
> Just click **Remove All** to remove the virtual camera, that should fix the problem.

Ok, had to reinstall it again to do so, but it seems to have fixed the issue. Thanks for your help.

**Comment by samozrejme** [comment]
Not sure if I should keep this open or not.
Since I got my answer but the issue can still occur for other users.

(github noob)

**Comment by hipersayanX** [comment]
>Not sure if I should keep this open or not.
>Since I got my answer but the issue can still occur for other users.

Yes, that's the right way since the bug is still ongoing.

**Comment by kianboon80** [comment]
i am also facing the same issue with portable version. How can i uninstall? I delete the folder away but problem still exist?

**Comment by hipersayanX** [comment]
@kianboon80 just do as said [here](https://github.com/webcamoid/webcamoid/issues/428#issuecomment-817238532).

**Comment by kianboon80** [comment]
@hipersayanX thank for your advise but I am unable to follow the instruction. When I open the app, it seems to initaite the laptop built-in cam and will shut down shortly after that.whole process in less than 2s. I am unable to access the app.

**Comment by hipersayanX** [comment]
@kianboon80 try [resetting Webcamoid](https://github.com/webcamoid/webcamoid/wiki/Reset-configs-and-starting-fresh#windows) and try again, if that does not work you will need to uninstall the virtual camera service manually.
Open an administrator cmd and run the following commands:

```
cd "C:\Program Files\AkVirtualCamera.plugin"
sc start AkVCamAssistant
regsvr32 /s /u .\x86\AkVirtualCamera.dll
regsvr32 /s /u .\x64\AkVirtualCamera.dll
sc stop AkVCamAssistant
.\x86\AkVCamAssistant  --uninstall
.\x64\AkVCamAssistant  --uninstall
```

**Comment by hipersayanX** [comment]
Problem solved in 9.0.0, closing.
